cmd/skaffold/app/cmd/debug.go

```diff
@@ -37,14 +46,70 @@ func NewCmdDebug() *cobra.Command {
 		WithLongDescription("Similar to `dev`, but configures the pipeline for debugging.").
 		WithCommonFlags().
 		WithHouseKeepingMessages().
```

Comment:
Maybe we should remove `WithHouseKeepingMessages()` for debug since we don't want update messages and such to show up on the output.

Reply:
I'm thinking it might make more sense to have a `skaffold filter --debugging` command.  We could then add `--build-artifacts` filtering there too.  WDYT @nkubala?

Reply:
sorry I missed this, looks like you decided on the `filter` subcommand which I think makes sense. makes it easier to extend this in the future if we need to as well